cart/Src/main.c

```diff
@@ -126,12 +132,10 @@ float Error_R=0;
 float Old_Error_L=0;
 float Old_Error_R=0;
 
-//volatile float Measured_Encoder_L;
-//volatile float Measured_Encoder_R;
-
 float Old_Motor_L;
 float Old_Motor_R;
 
+/************FOR PID*************/
```

Comment:
More description is necessary, such as PID control for cornering and avoiding obstacles.